client/index.html

```diff
@@ -9,32 +9,37 @@
 </head>
 
 <body>
-
+  <ul class="nav-horizontal">
+    <li><a href="#" >Logo</a></li>
+    <li><a href="#" data-bind="click: newNote">Add Note</a></li>
+    <li><a href="#">Clear Board</a></li>
+  </ul>
   <div>
-    <ul>
-      <li><label>Name:<input type="text" data-bind="textInput: name" /></label></li>
-      <li><label>Color:<input type="text" data-bind="textInput: color" /></label></li>
-      <li><button data-bind="click: newNote">New note</button></li>
-    </ul>
     <div class="board" data-bind="drop: true, foreach: notes">
-      <ul class="note" data-bind="drag: true, style: style">
-        <li><label>Title:<input type="text" data-bind="textInput: title" /></label></li>
-        <li><label>Content:<input type="text" data-bind="textInput: content" /></label></li>
-        <li><label>PosX:<input type="text" data-bind="textInput: posX" /></label></li>
-        <li><label>PosY:<input type="text" data-bind="textInput: posY" /></label></li>
+      <ul class="note" data-bind="drag: true, style: style" >
+        <!-- <li><label>Title:<input type="text" data-bind="textInput: title" /></label></li> -->
+        <li>
+            <h2  data-bind="jeditable: title, jeditableOptions: {}">Title</h2>

```

Comment:
I think that It's depends on whether you just have a unique text or a title and a description.